Simplenote/Classes/SPTagEntryField.m

```diff
@@ -22,15 +22,14 @@ - (instancetype)init
         self.textColor = [UIColor simplenoteTagViewTextColor];
         self.placeholdTextColor = [UIColor simplenoteTagViewPlaceholderColor];
         self.textAlignment = NSTextAlignmentNatural;
-        self.placeholder = NSLocalizedString(@"Add a tag...", nil);
+        self.placeholder = NSLocalizedString(@"Tag...", @"Placeholder text in textfield when adding a new tag to a note");
```

Comment:
If you want to keep "Tag", but fix the use of proper ellipsis character:
```suggestion
        self.placeholder = NSLocalizedString(@"Tag…", @"Placeholder text in textfield when adding a new tag to a note");
```

If you want to go with [my rationale](https://github.com/Automattic/simplenote-ios/pull/1443/files#r713792664) (since [current `develop` uses "Add a tag" in both `self.placeholder` and `[self setPlaceholder:]` calls](https://github.com/Automattic/simplenote-ios/blob/develop/Simplenote/Classes/SPTagEntryField.m#L25-L30) anyway)

```suggestion
        self.placeholder = NSLocalizedString(@"Add a tag…", @"Placeholder text in textfield when adding a new tag to a note");
```